src/vs/workbench/contrib/chat/browser/codeBlockPart.css

```diff
@@ -165,3 +172,8 @@
 .interactive-result-code-block.compare.no-diff .interactive-result-editor {
 	display: none;
 }
+
+.interactive-session .interactive-request .rendered-markdown .interactive-result-code-block .monaco-editor .overflow-guard,
+.interactive-session .interactive-request .rendered-markdown .monaco-editor {
+	width: 100% !important;
```

Comment:
I'm not sure this will do the right thing for a monaco-editor? They have to be properly sized in code, otherwise their layout will be broken